component_config/configRowSchema.json

```diff
@@ -159,15 +159,33 @@
             "title": "Personal data",
             "default": true,
             "propertyOrder": 8,
+            "format": "checkbox",
             "description": "Enable extraction of reviewer personal data (ID, name, URL, photo) and review identifiers for advanced customer analytics. Note: Personal data processing must comply with GDPR and other applicable data protection regulations. Ensure your data governance policies permit such collection before enabling."
         },
-        "incrementalOutput": {
-            "type": "boolean",
-            "title": "Incremental Loading",
-            "format": "checkbox",
-            "default": true,
-            "propertyOrder": 9,
-            "description": "If checked the component loads data incrementally otherwise all data from the component will be loaded."
+        "destination": {
+            "type": "object",
+            "title": "Destination",
+            "required": [
+                "outputTableName",
+                "incrementalOutput"
+            ],
+            "properties": {
+                "outputTableName": {
```

Comment:
Can you make this fied required and set default value for it. 
You can use google-maps-reviews as default.

If the field is empty the run in keboola failed.
![image](https://github.com/user-attachments/assets/5783233c-5319-434e-b747-9854f57a352e)